content/news/playtest-20160904.md

```diff
@@ -0,0 +1,68 @@
+Today we're happy to finally announce the first release candidate for our next OpenRA release.
+The new playtest contains over 1200 commits by 38 authors (including 11 first-time committers, thanks to all of you!).
+
+We rely on players to thoroughly test the playtest builds to catch any bugs or balance regressions before the final release, so head over to the [download page](/download/) to try it out today! If you encounter a bug, or have any feedback, then please create a ticket on our [Github issue tracker](http://bugs.open-ra.org), [find us on IRC](http://webchat.freenode.net/?channels=openra), create a topic [on our community forum](www.sleipnirstuff.com/forum/viewforum.php?f=80), or leave a message in the comments below.
+
+A lot of work in this release has focused on improving our core engine technology and paying down technical debt. The first place you will notice this is the mod chooser, which now understands the various install disks and digitial distributions of the original games. It can now automatically find and install all of the assets used by the mods, including the full motion videos and the additional music tracks included with the expansion disks.
+
+<div style="text-align:center" markdown="1">
+![New content installer](/images/news/20160904-cnc-contentinstall.png)
+</div>
+
+Because we have changed which assets are copied, all players will need to reinstall (or redownload) the assets for this playtest. These will be used for the future playtests and releases, so only needs to be done once.
+
+The new FMV installation support has arrived just in time (some would argue far too late!), because this playtest includes 7 more campaign missions for TD and one new campaign mission each for RA and D2K. RA also includes three custom multiplayer missions that can be selected from the mission chooser (for single-player) or the multiplayer lobby (for cooperative play).
```

Comment:
I'm never consistent with this myself, but at least in one sentence you should decide on whether to have a hyphen in (single|multi)-player or not ;)